### DISCUSSION TITLE
Keep only the latest app version when deployed

### DISCUSSION THREAD

**Discussion by HazardMK** [microsoft/AL-Go#1033] [Q&A] [answered]
When deploying an app using AL-GO-PTE to a sandbox env it adds increased version number (as it should be), but on every other deployment it increases the version and leaves the old ones in extension list with "**Is Installed**" = false. 
Is there a way to keep only the latest published version of the same extension in environment? Without manually unpublishing older versions.

**Comment by freddydk** [comment, marked as answer]
Today, it isn't possible, but... - in 25.4, there will be a new bound action in the automation API due to a partner contribution:

https://github.com/microsoft/ALAppExtensions/commit/d64f83940e832a897e8acc3c2d1ea8e4452711e2

When this is rolled out, we will add support for this in AL-Go for GitHub.
